install-timelord.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-if [ -z "$VIRTUAL_ENV" ]; then
+if [ -z "$VIRTUAL_ENV" ] && [ -z "${CHIATEST}" ] ; then
```

Comment:
do you not create a venv inside the docker image? but you istall everything to the image's system?